tests/t147_event_sdt.py

```diff
@@ -15,5 +15,10 @@ def __init__(self):
    3.017 us [28141] | } /* main */
 """)
 
+    def build(self, name, cflags='', ldflags=''):
+        if TestBase.get_machine(self) != 'x86_64':
+            return TestBase.TEST_SKIP
+        return TestBase.build(self, name, cflags, ldflags)
+
```

Comment:
Like in the dynamic tracing, can you please put some generic function in the TestBase so that we can easily change the condition once we have the support on the other architectures?

Reply:
I split `check_arch_sdt_support` and `check_arch_mfentry_mnop_mcount_support` differently. Here, `check_arch_mfentry_mnop_mcount_support` expects https://github.com/namhyung/uftrace/pull/1287/commits/78ea4b25728176de8feaf9211fd517b9a5e8c989 to be merged.